### PR TITLE
Test rails 4.2

### PR DESCRIPTION
**Pull request by jcoyne**

### CODE REVIEW

**Comment by scande3** [comment]
Looks good to me - merging and closing this.

File: .travis.yml [hunk 1]
```diff
@@ -1,18 +1,19 @@
 language: ruby
+cache: bundler
+sudo: false
 rvm:
-  - 2.0
-  - 2.1
```
Comment (dbrower) [inline .travis.yml]: Why remove testing for older rubies?

**Comment by dbrower** [comment]
I'm having trouble running bundler with this PR. I get the following error

```
Bundler could not find compatible versions for gem "railties":
  In Gemfile:
    responders (~> 2.0) ruby depends on
      railties (< 5, >= 4.2.0.alpha) ruby

    rspec-rails (>= 0) ruby depends on
      railties (4.0.0)
```

File: .travis.yml (hunk 1 above)
Comment (jcoyne) [inline .travis.yml]: Just because I don't see why this is necessary. If you want to re-add go ahead.

**Comment by jcoyne** [comment]
@dbrower do you already have a Gemfile.lock? Did you do "bundle update" or "bundle install"?

**Comment by dbrower** [comment]
@jcoyne I was doing bundle install, and it was a stale Gemfile.lock. Deleting the file fixed it.

**Pull request merged**